lib/tasks/export_exercise_collection.rake

```diff
@@ -0,0 +1,19 @@
+namespace :codeharbor do
+
+  desc 'write displaynames retrieved from the account service as csv into the codeocean database'
+
+  task :export_exercise_collection, [:exercise_collection_id, :push_url, :push_token] => [:environment] do |t, args|
+
+    begin
+      ExerciseCollection.find(args.exercise_collection_id).exercises.each do |exercise|
+        oauth2Client = OAuth2::Client.new('client_id', 'client_secret', :site => args.push_url)
```

Comment:
still hard coded strings here

Reply:
I think they are okay here, since we only use rake tasks internally and client_id and client_secret are only there because they are required by OAuth2, but are not used otherwise.
What do you think?